sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/util/SearchResult.java

```diff
@@ -0,0 +1,26 @@
+package com.exadel.sandbox.team5.dao.util;
+
+import org.springframework.data.domain.PageRequest;
+
+import java.awt.print.Pageable;
+import java.util.List;
+
+public class SearchResult {
+    private int totalCount;
+    private PageRequest result;
```

Comment:
Does not make sense to use PageRequest here. We need field for page response, ``` List<T>``` or object of PageResponse class

Reply:
Agree. Please use `SearchResult` with generic parameter T

Reply:
Done.